types/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for Chalk
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export = chalk;
+
+declare function chalk(...text: string[]): string;
+declare function chalk(text: TemplateStringsArray, ...placeholders: string[]): string;
+
+declare namespace chalk {
+
+	export enum ChalkLevel {
+		Disabled = 0,
+		Basic = 1,
+		Extended = 2,
+		TrueColor = 3
+	}
+
+	interface ChalkConstructorOptions {
+		enabled?: boolean;
+		level?: ChalkLevel;
+	}
+
+	export function constructor(options?: ChalkConstructorOptions): typeof chalk;
+
+	export let enabled: boolean;
```

Comment:
chalk is imported as an ES6 module where this is disallowed. If it is imported via `import chalk = require('..');` it is possible to set those values. I can change the test if it is wanted.